Gemfile

```diff
@@ -4,10 +4,10 @@ git_source(:github) do |repo_name|
   repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
   "https://github.com/#{repo_name}.git"
 end
-ruby '2.5.1'
+ruby '2.5.3'
```

Comment:
rubyのバージョン変更は避けたほうが良いです
railsのバージョンのみ変更することが可能です

You should avoid changing the version of ruby.
It is possible to change only the version of rails.

Reply:
Is there a reason to keep it at 2.5.1? Every time I try to run
`bundle update`
I get the following:
Your Ruby version is 2.5.3, but your Gemfile specified 2.5.1
Due to difficulty with using rvm on git bash, it is difficult for me to get the 2.5.1 version installation.

Reply:
できればrvmかrbenvを使って欲しかったです
チーム開発で、コア言語の必須でないアップデートを頻繁に行うのは好ましくありません
ですがこれはテストなので、難しいのなら許容します

I wanted to use rvm or rbenv if possible.
In team development, it is not reasonable to frequently make nonessential updates of the core language.
But this is a test, so it will be acceptable if it is difficult.